R/modify-model-field.R

```diff
@@ -164,7 +170,8 @@ replace_all_tags <- function(.mod, .tags) {
     .field = YAML_TAGS,
     .value = .tags,
     .append = FALSE,
-    .char_value = TRUE
+    .char_value = TRUE,
+    .bool_value = FALSE
```

Comment:
Specifying optional args is nice for guarding against flips in the default, but that is very unlikely to happen in this case, and you don't do it for some other `modify_model_field` calls.  I'd suggest not bothering adding the `.bool_value = FALSE` to any functions.